run.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+from aiproxy.app import create_app
+
+app = create_app()
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Comment:
run.py is nice just because its conventional / frequently present in flask apps. Flask isn't big on convention, really, but this is a common one, and combined with a setup.py will allow a lot of python devs to "run the app" without a lot of reading.